src/python/pants/source/source_root.py

```diff
@@ -145,6 +145,8 @@ class SourceRootConfig(Subsystem):
 
     options_scope = "source"
 
+    DEFAULT_ROOT_PATTERNS = ["/", "src", "src/python", "src/py"]
```

Comment:
This is very Python centric. We're okay with that because of the 2.0 release, I take it?

Reply:
I figured that we could fairly easily backwards-compatibly add a "languages" option or union later, which would have the effect of making this computed?

Reply:
Yeah, we can add specific languages here as we support them.